musicplatform/user/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from .models import User
+
+admin.site.register( User )
```

Comment:
you didn't modify the admin to show the bio as a textarea. first, you should creat a custom form in `forms.py`:
```python
class CustomUserChangeFrom(UserChangeForm):
    class Meta(UserChangeForm.Meta):
        model = User
        fields = UserChangeForm.Meta.fields
        widgets = {
            'bio': forms.Textarea
        }
```

and use this form in the `admin.py`:
```python
class CustomUserAdmin(UserAdmin):
    form = CustomUserChangeFrom
    model = User
    list_display = ['username', 'email', 'first_name', 'last_name', 'is_staff', 'bio']
    fieldsets = UserAdmin.fieldsets + (
        ("User's Bio", {'fields': ('bio',)}),
    )
```